templates/projects/webbasic/project.json

```diff
@@ -48,9 +48,7 @@
       "npm install",
       "bower install",<% if(!grunt){ %>
```

Comment:
The comma needs to be removed if `!grunt` evaluates to true as it is left otherwise:
```
"scripts": {
    "prepublish": [
      "npm install",
      "bower install",
    ]
  }
```
The trailing comma is OK in syntax, but is flagged by linters (e.g. in VSCode as `value expected`)